src/services/CertificateGenerationService.ts

```diff
@@ -162,6 +162,16 @@ class CertificateGenerationService {
    * @param testResult - source test result for certificate generation
    */
   public async generatePayload(testResult: any) {
+    let name = testResult.testerName;
+
+    const nameArrayList: string[] = name.split(",");
+
+    if (nameArrayList.length === 2) {
+      [nameArrayList[0], nameArrayList[1]] = [nameArrayList[1], nameArrayList[0]];
+      name = name.split(", ").reverse().join(", ");
```

Comment:
Please check the AC's with a BA.  This code will return "Melanie, Coslett" which matches what the AC's state the logic should be, but not the example.  I don't think there should be a comma between the first name(s) and surname.